functions/go/set-labels/transformer/labels.go

```diff
@@ -94,26 +101,147 @@ func (p *LabelTransformer) setLabelsInSpecs(o *fn.KubeObject) error {
 func (p *LabelTransformer) Transform(objects fn.KubeObjects) error {
```

Comment:
func Transform() {
     // filter unrelated resources. like localconfig or Kptfile metaconfig
     err := func() {
       // set meta labels
       UpdateLabelsInMeta
       // set pod labels 
       UpdatePodLabels 
       // labelSelector
       ...
       // optional (Not CreateifNot exist)  
     }()
     if err != nil {
         LogError
     }
     LogSummary
}